src/pools-helper.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  LoggerConstructor,
+  PoolLiquidity,
+  Logger,
+  Token,
+  Address,
+} from './types';
+import { DexAdapterService } from './dex';
+
+export class PoolsHelper {
+  logger: Logger;
+
+  constructor(
+    protected dexAdapterService: DexAdapterService,
+    loggerConstructor: LoggerConstructor,
+  ) {
+    this.logger = loggerConstructor(`PoolsHelper_{dexAdapterService.network}`);
```

Comment:
Missing the `$` in the string interpolation.

Reply:
Fixed